.goreleaser.yml

```diff
@@ -29,7 +32,10 @@ builds:
   goos:
     - linux
   goarch:
-    - amd64    
+    - amd64 
+  hooks:
+    pre: sed -i s/~mailchain-version~/{{.Version}}/ ./cmd/mailchain/internal/http/handlers/swagger.go
```

Comment:
Since the builds were getting sequentially (atleast during make snapshot) and expecting the same in prod as well, I could have not duplicated the same command multiple times for every build. But adding it, just so if they are parallelised in the future.